### PR TITLE
NEXT-25207 - Remove development psh commands reference and use composer scripts instead

### PR DESCRIPTION
**Pull request by keulinho**

### CODE REVIEW

File: guides/plugins/plugins/testing/end-to-end-testing.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ In end-to-end testing \(E2E testing in short\) real user workflows are simulated
 
 To use Shopware E2E tests, at first you need to have a Shopware 6 installation running. Making sure, that your tests are reliable, you should have a clean installation. Cleanup means no categories, no products, no settings, nothing!
 
-The easiest way to clean up your installation is the initialization. Using the command `./psh.phar init` Shopware 6 gets initialized clean and without demo data. Installation of E2E dependencies can be accomplished separately by running `npm install` in the E2E folder you're using, e.g. for Shopware Administration it's `src/Administration/Resources/app/administration/test/e2e`.
+The easiest way to clean up your installation is the initialization. Using the command `composer run init` Shopware 6 gets initialized clean and without demo data. Installation of E2E dependencies can be accomplished separately by running `npm install` in the E2E folder you're using, e.g. for Shopware Administration it's `src/Administration/Resources/app/administration/test/e2e`.
```
Comment (keulinho) [inline guides/plugins/plugins/testing/end-to-end-testing.md:11]: I'm not sure with all the docker mentioned, what to execute outside or inside and if it is even needed anymore cc @leichteckig can you please check that?

File: guides/plugins/apps/local-development/app-development-with-docker.md [hunk 2]
```diff
@@ -1,216 +0,0 @@
-# App development with docker
```
Comment (keulinho) [inline guides/plugins/apps/local-development/app-development-with-docker.md:1]: removed it as we don't support that anymore without the dev template

File: guides/plugins/plugins/administration/add-custom-field.md [hunk 3]
```diff
@@ -71,18 +71,17 @@ This block contains the whole settings form of the product detail page. In order
 As mentioned above, Shopware 6 is looking for a `main.js` file in your plugin. Its contents get minified into a new file named after your plugin and will be moved to the `public` directory of Shopware 6 root directory. Given this plugin would be named "AdministrationNewField", the minified javascript code for this example would be located under `<plugin root>/src/Resources/public/administration/js/administration-new-field.js`, once you run the command following command in your shopware root directory:
 
 {% tabs %}
-{% tab title="Development template" %}
+{% tab title="Production template" %}
```
Comment (shyim) [inline guides/plugins/plugins/administration/add-custom-field.md:74]:
```suggestion
{% tab title="Template" %}
```

File: guides/plugins/plugins/administration/add-custom-module.md [hunk 4]
```diff
@@ -200,18 +200,18 @@ Its contents get minified into a new file named after your plugin and will be mo
 Given this plugin would be named "AdministrationNewModule", the bundled and minified javascript code for this example would be located under `<plugin root>/src/Resources/public/administration/js/administration-new-module.js`, once you run the command following command in your shopware root directory:
 
 {% tabs %}
-{% tab title="Development template" %}
+{% tab title="Production template" %}
```
Comment (shyim) [inline guides/plugins/plugins/administration/add-custom-module.md:203]:
```suggestion
{% tab title="Template" %}
```

File: guides/plugins/plugins/administration/add-custom-field.md [hunk 5]
```diff
@@ -71,18 +71,17 @@ This block contains the whole settings form of the product detail page. In order
...
 
 ```bash
-./psh.phar administration:build
+./bin/build-administration.sh
 ```
 
 {% endtab %}
-
-{% tab title="Production template" %}
+{% tab title="platform only" %}
```
Comment (shyim) [inline guides/plugins/plugins/administration/add-custom-field.md:81]:
```suggestion
{% tab title="platform only (contribute)" %}
```

File: guides/plugins/plugins/administration/add-custom-module.md [hunk 6]
```diff
@@ -200,18 +200,18 @@ Its contents get minified into a new file named after your plugin and will be mo
...
 
 ```bash
-./psh.phar administration:build
+./bin/build-administration.sh
 ```
 
 {% endtab %}
 
-{% tab title="Production template" %}
+{% tab title="platform only" %}
```
Comment (shyim) [inline guides/plugins/plugins/administration/add-custom-module.md:211]:
```suggestion
{% tab title="platform only (contribute)" %}
```

**Review by shyim — CHANGES_REQUESTED** [review]

File: guides/plugins/plugins/administration/add-custom-field.md (hunk 5 above)
Comment (keulinho) [reply guides/plugins/plugins/administration/add-custom-field.md]: i use `platform only (contribution setup)` now

**Review by Christian-Rades — APPROVED** [review]

**Review by shyim — APPROVED** [review]

**Pull request merged**